### PR TITLE
Feature/new files

### PR DESCRIPTION
**Pull request by andriikamaldinov1**

### CODE REVIEW

**Review by lachesse — APPROVED** [review]

File: chip/chip.component.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export class ChipComponent {
     @Input()
     public chipBgColor = 'rgba(0,0,0,0.05)';
     @Input()
-    public chipTextColor = '#191919';
+    public chipTextColor = '';
```
Comment (xakeppok) [inline chip/chip.component.ts:15]: why did you remove default text? did you change it everywhere?

Comment (andriikamaldinov1) [reply chip/chip.component.ts:15]: default text used only in open source, on website its color white!
